server/game/cards/events/01/thehandsjudgment.js

```diff
@@ -28,7 +28,7 @@ class TheHandsJudgment extends DrawCard {
             return super.getCost();
         }
 
-        return this.eventToInterrupt.getCost();
+        return this.eventToInterrupt.cardData.cost || 0;
```

Comment:
Oh. Ew. Which idiot did that 🙃

In that case we probably need a getPrintedCost as for strength